addons/edi/models/edi_document.py

```diff
@@ -71,46 +71,40 @@ def _default_project_id(self):
 
     # Sequence for generating document names
     sequence_id = fields.Many2one(
-        "ir.sequence",
-        string="Document Name Sequence",
-        required=True,
-        default=_default_sequence_id,
+        "ir.sequence", string="Document Name Sequence", required=True, default=_default_sequence_id
     )
 
     # Issue tracker used for asynchronously reporting errors
     project_id = fields.Many2one(
-        "project.project",
-        string="Issue Tracker",
-        required=True,
-        default=_default_project_id,
+        "project.project", string="Issue Tracker", required=True, default=_default_project_id
     )
 
     # Control visibility in the UI.
     active = fields.Boolean(
         default=True, string="Active", help="Display in list views or searches."
     )
 
+    # Control behaviour if an error occurs.
+    fail_fast = fields.Boolean(
+        string="End document execution immediately on error",
```

Comment:
Can we just call this `Fail Fast`? The help string can have the extra detail.

Reply:
@PeterC10 Done